apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/pathAssertions.kt

```diff
@@ -147,3 +147,24 @@ val <T : Path> Expect<T>.parent get(): Expect<Path> = ExpectImpl.path.parent(thi
  */
 fun <T : Path> Expect<T>.parent(assertionCreator: Expect<Path>.() -> Unit): Expect<T> =
     ExpectImpl.path.parent(this).addToInitial(assertionCreator)
+
+/**
+ * Creates an [Expect] for the extension of this Path so that further fluent calls are assertions about it.
+ *
+ * @return The newly created [Expect].
+ *
+ * @since 0.9.0
+ */
+val <T : Path> Expect<T>.extension get(): Expect<String> =
+    ExpectImpl.path.extension(this).getExpectOfFeature()
+
+/**
+ * Expects that this [Path] has a given extension.
```

Comment:
please copy KDOC from `fileName` with assertionCreator above and adapt

Reply:
done